packages/microsoft-reactnative-sampleapps/windows/SampleAppCPP/App.cpp

```diff
@@ -19,11 +19,19 @@ App::App() noexcept {
   MainComponentName(L"SampleApp");
   JavaScriptMainModuleName(L"index.windows");
 
-#if _DEBUG
+#if BUNDLE
```

Comment:
Maybe define macros to make this more concise?
```
#if BUNDLE
#define ARG1 L"V1"
#define ARG2 false
#else
#define ARG1 L"V2"
#define ARG2 true
#endif

func1(ARG1);
func2(ARG2);

Reply:
I will not make change. In our template, C++ and C# follows the same pattern.

C++:
```
#if BUNDLE
    JavaScriptBundleFile(L"index.windows");
    InstanceSettings().UseWebDebugger(false);
    InstanceSettings().UseLiveReload(false);
#else
```

C#
```
#if BUNDLE
        JavaScriptBundleFile = "index.windows";
        InstanceSettings.UseWebDebugger = false;
        InstanceSettings.UseLiveReload = false;
#else
```